responsive-test-utils/src/test/java/dev/responsive/kafka/api/ResponsiveTopologyTestDriverKeyValueStoreTest.java

```diff
@@ -98,32 +109,36 @@ public void shouldRunAllKVStoreTypesWithTTLWithoutResponsiveConnection(final KVS
         "output", new StringDeserializer(), new StringDeserializer());
 
     // When:
-    people.pipeInput("1", "1,alice,CA"); // time = 0
-    driver.advanceWallClockTime(Duration.ofMillis(5));
```

Comment:
This test now reflects how a regular Streams app would function and how a regular TTD test would be written